Parse/src/main/java/com/parse/Parse.java

```diff
@@ -46,6 +110,8 @@
   private static boolean isLocalDatastoreEnabled;
   private static OfflineStore offlineStore;
 
+  private static Configuration configuration;
```

Comment:
I don't think we need to hold this, we can just pass in `configuration.interceptors` through to `initializeParseHttpClientsWithParseNetworkInterceptors()`